lib/objects/objectsInMemFileDB.js

```diff
@@ -904,11 +904,11 @@ class ObjectsInMemoryFileDB extends InMemoryFileDB {
                             fs.unlinkSync(location);
                         }
                     }
-                    const path = path.dirname(f);
+                    const filepath = path.dirname(f);
```

Comment:
Nitpick, but: `filePath` with the same capitalization

Reply:
yw ;-)